tg.js

```diff
@@ -308,12 +308,12 @@ settings.gameprivacy.header Pagkapribado ng Laro
 settings.gameprivacy.streamer Mode ng Streamer
 settings.gameprivacy.anon Anonymous Mode
 settings.gameprivacy.hideGameTitle Itago ang Pamagat ng Laro
-settings.gameprivacy.showRegion Show Region Info
+settings.gameprivacy.showRegion Ipakita ang Detalye ng Rehiyon
 settings.gameprivacy.verified Ipakita: Verification Badge
 settings.gameprivacy.premium Ipakita: Premium Badge
 settings.gameprivacy.partner Ipakita: Partnered Badge
 settings.gameprivacy.rank Ipakita: Ranked Badge
-settings.gameprivacy.custom Ipakita: Custom Badge
+settings.gameprivacy.custom Ipakita: Kustom Badge
```

Comment:
```suggestion
settings.gameprivacy.custom Ipakita: Custom Badge
```
please revert, there's no need to change the c in custom on this line when there are 100 other lines with 'Custom' and not 'Kustom'